### PR TITLE
updated heroku-client and migrate to async/await

### PR DESCRIPTION
**Pull request by jdxcode**
depends on https://github.com/heroku/heroku-redis-jsplugin/pull/49

### CODE REVIEW

**Comment by jdxcode** [comment]
@uhoh-itsmaciek would you mind taking a look at these PRs? This plugin has some pretty old dependencies so I'd like to update them to streamline the core

File: lib/shared.js [hunk 1]
```diff
@@ -1,109 +1,104 @@
...
-  })
 
-  // print out the info of the addon and redis db info
-  for (let db of databases) {
-    if (db.redis === null) {
-      continue
-    }
+      // print out the info of the addon and redis db info
+      for (let db of databases) {
+        const redis = await db.redis
```
Comment (uhoh-itsmaciek) [inline lib/shared.js]: Oh I see--you're `awai`ting just-in-time here instead, right? In which case you don't need the `await` above at all?

Comment (jdxcode) [reply lib/shared.js]: Yeah you're right, I'll remove that extra one

**Comment by uhoh-itsmaciek** [comment]
@jdxcode have you run the updated commands? if not, i'd like to run through them quickly as a smoke test before we ship this just to make sure

**Comment by jdxcode** [comment]
ok this is ready to go I think @uhoh-itsmaciek

**Pull request merged**